source/administration/minio-console.rst

```diff
@@ -11,34 +11,40 @@ MinIO Console
    :depth: 2
 
 
-The MinIO Console is a rich graphical user interface that provides similar
-functionality to the :mc:`mc` command line tool.
+The MinIO Console is a rich graphical user interface that provides similar functionality to the :mc:`mc` command line tool.
 
 .. image:: /images/minio-console/minio-console.png
    :width: 600px
    :alt: MinIO Console Landing Page provides a view of Buckets on the deployment
    :align: center
 
-You can use the MinIO Console for administration tasks like Identity and 
-Access Management, Metrics and Log Monitoring, or Server Configuration.
+You can use the MinIO Console for administration tasks like Identity and Access Management, Metrics and Log Monitoring, or Server Configuration.
 
-The MinIO Console is embedded as part of the MinIO Server binary starting 
-with :minio-release:`RELEASE.2021-07-08T01-15-01Z`. You can also deploy a 
-standalone MinIO Console using the instructions in the 
-:minio-git:`github repository <console>`.
+The MinIO Console is embedded as part of the MinIO Server. 
+You can also deploy a standalone MinIO Console using the instructions in the :minio-git:`github repository <console>`.
 
-You can explore the Console using https://play.min.io:9443. Log in with
-the following credentials:
+Logging into the MinIO Console depends on how you configured identity management for the deployment.
 
-- Username: ``Q3AM3UQ867SPQQA43P2F``
-- Password: ``zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG``
+- When using the built-in MinIO identity management solution, the sign-in screen displays a standard login screen.
+  Enter your Username and Password to log in to the MinIO Console.
+  Or, you can use a :ref:`MinIO's Security Token Service (STS) <minio-security-token-service>` credential by selecting :guilabel:`Use STS` and supplying the Username, Secret and Token.
+- If the deployment uses a third-party identity provider solution, select the button to proceed to the login screen.
```

Comment:
I think you could probably merge these two? e.g. "For deployments configured to use a third-party identity provider, select the :guilabel:`use STS` button to log in with the externally managed credentials.

Reply:
Except it's four different descriptions of what the user might do:

1. Authenticate with a MinIO identity
2. Authenticate with a 3rd-party app using an STS
3. Log in with the configured OpenID or LDAP provider
4. Choose from one of several configured OpenID and/or LDAP providers

Reworked some to hopefully clarify better.